diff -ru bglibs-1.106.orig/Makefile bglibs-1.106/Makefile
--- bglibs-1.106.orig/Makefile	2009-02-19 18:11:51.000000000 +0100
+++ bglibs-1.106/Makefile	2013-09-12 15:11:46.887246484 +0200
@@ -145,8 +145,8 @@
 	) >compile
 	chmod 755 compile
 
-crc-gentab: crc-gentab.o ltload libbg-msg.la libbg-iobuf.la libbg-sysdeps.la
-	./ltload crc-gentab libbg-msg.la libbg-iobuf.la libbg-sysdeps.la -static 
+crc-gentab: crc-gentab.o ltload libbg-msg.la libbg-iobuf.la libbg-fmt.la libbg-str.la libbg-sysdeps.la
+	./ltload crc-gentab libbg-msg.la libbg-iobuf.la libbg-fmt.la libbg-str.la libbg-sysdeps.la -static 
 
 crc-gentab.o: compile crc-gentab.c sysdeps.h iobuf/iobuf.h iobuf/common.h iobuf/ibuf.h iobuf/obuf.h msg/msg.h msg/debug.h
 	./compile crc-gentab.c
